apps/modernization-ui/src/components/ModalComponent/ModalComponent.scss

```diff
@@ -0,0 +1,39 @@
+@import 'settings.scss';
+
+.usa-js-modal--active {
+    .usa-modal__heading {
+        padding: 1.5rem;
+        margin-bottom: 0;
+        border-bottom: 1px solid $color-bg-base-lightest;
+    }
+    .usa-modal__footer {
+        display: flex;
+        flex-direction: row;
+        gap: 0.8rem;
+        justify-content: flex-end;
+        padding: 1.5rem;
+        border-top: 1px solid $color-bg-base-lightest;
+    }
+}
+.usa-modal {
+    &.tall {
+        min-height: 50rem;
+        height: 50rem;
```

Comment:
Probably a question for design team: What about specifying a % height here instead of 50rem. This causes the user to have to scroll if their screen size is too small.